problems/arraysWithLoops.js

```diff
@@ -66,15 +113,27 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
-
+function findSecondSmallest(nums) {
+  let secondSmallest = nums[0];
+  for(let i = 0; i <= nums.length; i++){
+    if (nums[i] < secondSmallest) secondSmallest = nums[i];
+  }
+  return secondSmallest + 1
```

Comment:
Same as before, this is only correct because of the specific test cases. 

We want our function to work no matter what numbers are in the array